datafusion/sql/src/unparser/expr.rs

```diff
@@ -711,6 +718,10 @@ mod tests {
                 Expr::IsUnknown(Box::new((col("a") + col("b")).gt(lit(4)))),
                 r#"(("a" + "b") > 4) IS UNKNOWN"#,
             ),
+            (
+                Expr::Not(Box::new(col("a"))),
```

Comment:
I think you could write this more concisely using https://docs.rs/datafusion/latest/datafusion/logical_expr/fn.not.html

So something like `not(col(a))`